inject.js

```diff
@@ -527,11 +540,58 @@
           setMark(v);
         } else if (action === 'jump') {
           jumpToMark(v);
+        } else if (action === 'adaptive') {
+          toggleAdoptiveSpeed(v, controller);
         }
       }
     });
   }
 
+  function toggleAdoptiveSpeed(v, controller) {
```

Comment:
I think you may have meant to name this `toggleAdaptiveSpeed`

Reply:
You are right.